packages/babel-parser/src/parser/expression.js

```diff
@@ -2199,6 +2208,7 @@ export default class ExpressionParser extends LValParser {
 
   parsePropertyName(
     prop: N.ObjectOrClassMember | N.ClassMember | N.TsNamedTypeElementBase,
+    refExpressionErrors?: ?ExpressionErrors,
```

Comment:
Can you add a comment that passing `refExpressionErrors` means "private names are allowed"?

Reply:
Though currently `refExpressionErrors` here means "private names are allowed", the `refExpressionErrors` tracks those language features that are valid in pattern but invalid in expressions. When we are passing `refExpressionErrors`, it means we are handling ambiguous syntax that should only be determined when we check expression errors later. 

In this way the `refExpressionErrors` is the opposite of expression scope, which tracks those valid in expression but invalid in patterns.